---
title: C28125
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28125
helpviewer_keywords:
- C28125
ms.assetid: 3f7b8db4-6a64-4480-919e-3f3ceca7f948
author: mikeblome
ms.author: mblome
manager: markl
ms.workload:
- multiple
ms.openlocfilehash: 82c4997c1069d6d8a41cf9be56445ce089b93d15
ms.sourcegitcommit: 535ef05b1e553f0fc66082cd2e0998817eb2a56a
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 10/07/2019
ms.locfileid: "72010523"
---
# <a name="c28125"></a>C28125
Warnung C28125: Die Funktion muss innerhalb eines try/with-Blocks aufgerufen werden.

 Der Treiber ruft eine Funktion auf, die innerhalb eines try/with-Blocks aufgerufen werden muss, z. b. [Probe forread](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforread), [probeforwrite](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-probeforwrite), [MmProbeAndLockPages](https://docs.microsoft.com/windows-hardware/drivers/ddi/content/wdm/nf-wdm-mmprobeandlockpages).

## <a name="example"></a>Beispiel
 Im folgenden Codebeispiel wird diese Warnung generiert:

```
ProbeForRead(addr, len, 4);
```

 Im folgenden Codebeispiel wird diese Warnung vermieden:

```

__try
{
    ProbeForRead(addr, len, 4);
}
__except(EXCEPTION_EXECUTE_HANDLER)
{
    Status = GetExceptionCode();
    ... report error status
}
```